release/smoke-tests/run-smoke-tests.sh

```diff
@@ -82,21 +89,34 @@ while getopts "hv:r:o::" arg; do
         o)
             export OPENSEARCH_VERSION=$OPTARG
             ;;
+        i)
+            export IMAGE_NAME=$OPTARG
+            ;;
         ?)
             echo "Invalid option: -${arg}"
             end_tests 1
             ;;
     esac
 done
 
-if ! docker pull "${REPOSITORY}/data-prepper:${TAG_NAME}" > /dev/null
+if [ -z ${REPOSITORY+x} ]
+then
+    # REPOSITORY not defined
+    export DOCKER_IMAGE="${IMAGE_NAME}:${TAG_NAME}"
+else
+    # REPOSITORY is defined
+    export DOCKER_IMAGE="${REPOSITORY}/${IMAGE_NAME}:${TAG_NAME}"
+    docker pull "${DOCKER_IMAGE}" > /dev/null 2> /dev/null
+fi
+
+if ! docker inspect --type=image "${DOCKER_IMAGE}" > /dev/null
 then
     echo "--------------------------------------------------------------------------"
-    echo "Unable to pull image \"${REPOSITORY}/data-prepper:${TAG_NAME}\" are you sure it exists?"
+    echo "Unable to pull image \"${DOCKER_IMAGE}\" are you sure it exists?"
```

Comment:
This error message will be inaccurate for a local image which does not exist. An alternative could be: `Unable to find image \"${DOCKER_IMAGE}\" are you sure it exists?`